product_docs/docs/pem/8/pem_query_tool.mdx

```diff
@@ -137,19 +137,19 @@ On successful generation of `Explain` plan, it will create three tabs/panels und
 
 ### Graphical tab
 
-Hover over an icon on the `Graphical` tab to review information about that item; a popup window will display information about the selected object. For information on JIT statistics, triggers and a summary, hover over the icon on top-right corner; a similar popup window will be displayed when appropriate.
+Hover over an icon on the `Graphical` tab to review information about that item; a popup window displays information about the selected object. For information on JIT statistics, triggers, and a summary, hover over the icon on top-right corner; a similar popup window is displayed when appropriate.
 
 Please note that `EXPLAIN VERBOSE` cannot be displayed graphically.
 
-Use the download button on top left corner of the `Explain` canvas to download the plan as an SVG file. Please note that `Download as SVG` feature is not supported on Internet Explorer.
+Use the download button on the top left corner of the `Explain` canvas to download the plan as an SVG file. Please note that `Download as SVG` feature is not supported on Internet Explorer.
```

Comment:
Is the download button a named button? We don't use the word "button." Select Download? No tagging on the canvas name. We never use. "Please." We also don't use "Note that" unless we use a note. We are also using contractions. Put goal before action.

To download the plan as an SVG file, select **Download** in the top-left corner of the Explain canvas. **Download as SVG** isn't supported on Internet Explorer.